psi4/src/psi4/libfunctional/LibXCfunctional.cc

```diff
@@ -187,7 +191,7 @@ std::shared_ptr<Functional> LibXCFunctional::build_worker() {
 
     // User tweakers
     if (user_tweakers_.size()) {
-        func->set_tweak(user_tweakers_);
+        func->set_tweak(user_tweakers_, true);
```

Comment:
Please remind me, do we have a test checking that this still works?

Reply:
not a test where the user is building a tweaked fctl, but all the dft_builder fctls run through this code.